src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -64,24 +65,19 @@
               (assoc-in [:wallet :ui :send :to-address] to-address)
               (assoc-in [:wallet :ui :send :address-prefix] prefix)
               (assoc-in [:wallet :ui :send :selected-networks] selected-networks))
-      :fx [[:dispatch
-            [:navigate-to-within-stack
-             (if token?
-               [:wallet-send-input-amount stack-id]
-               [:wallet-select-asset stack-id])]]]})))
```

Comment:
I think we should just replace these events dispatch with `:navigation/wizard` calls (which will then lead to a navigate event) instead of removing it. WDYT @J-Son89 @mmilad75?

Reply:
perhaps you could provide a code snippet example of how you see that looking @briansztamfater ?

Reply:
Sure, I believe that something like this should work. In that way, we only modify navigation logic, and `:navigation/wizard` would decide to which screen to navigate next.

```
(rf/reg-event-fx
 :wallet/select-send-address
 (fn [{:keys [db]} [{:keys [address token? recipient]}]]
   (let [[prefix to-address] (utils/split-prefix-and-address address)
         test-net?           (get-in db [:profile/profile :test-networks-enabled?])
         goerli-enabled?     (get-in db [:profile/profile :is-goerli-enabled?])
	@@ -64,24 +65,19 @@
              (assoc-in [:wallet :ui :send :to-address] to-address)
              (assoc-in [:wallet :ui :send :address-prefix] prefix)
              (assoc-in [:wallet :ui :send :selected-networks] selected-networks))
      :fx [[:dispatch
            **[:navigation/wizard
                 {:current-screen :wallet-select-address
                  :flow-config       send-flow-config}]]]})))**
```

Reply:
For `:wallet/send-select-token`, would be

```
(rf/reg-event-fx :wallet/send-select-token
 (fn [{:keys [db]} [{:keys [token]}]]
   {:db (-> db
            (update-in [:wallet :ui :send] dissoc :collectible)
            (assoc-in [:wallet :ui :send :token] token))
    :fx [[:dispatch [:wallet/clean-suggested-routes]]
         [:dispatch [:navigation/wizard {:current-screen :wallet-select-address
                                                              :flow-config       send-flow-config}]]]}))
```

Reply:
In this case, we still need to call the previous event which is `:wallet/select-send-address` in the screen (and the same for other screens). Both cases seem ok to me @J-Son89 @briansztamfater

Reply:
@mmilad75 Sounds good to me, IMO that will make the approach simpler and the code easier to understand